src/components/Forms/AdditionalContactForm.vue

```diff
@@ -250,7 +255,13 @@ export default {
     font-size: 1rem;
     margin-left: 6px;
   }
-
+  .input-datepicker {
+    font-size: 10px;
+  }
+  .input-datepicker::v-deep input {
```

Comment:
❓ What's v-deep?

Reply:
Long story...

Basically "scoped" in vue stops us from using the > selector. What I wanted to write was something along the lines of:

```css
.input-datepicker > input {
    ...
}
```
But it didnt like that, turns out you need to use the v-deep selector to get around that.

Vue bad. 😠